.github/workflows/main.yml

```diff
@@ -9,9 +9,8 @@ on:
 
 env:
   REGISTRY: ghcr.io
-  IMAGE_NAME: ghcr.io/${{ github.repository }}
+  IMAGE_NAME: ghcr.io/genomicdatainfrastructure/gdi-userportal-frontend
   AZURE_WEBAPP_NAME: catalogue-test
-  DOCKER_METADATA_OUTPUT_TAGS:
```

Comment:
`DOCKER_METADATA_OUTPUT_TAGS` and `IMAGE_NAME` are fundamentaly different, as the first contains the tag we want to deploy, as doesn't.

I would suggest you to fix every place with `"${{ env.DOCKER_METADATA_OUTPUT_TAGS }}"` with `"${{ env.IMAGE_NAME }}:main"`.

And don't forget to replace all references of `DOCKER_METADATA_OUTPUT_TAGS`.